tests/test_getter_setter.py

```diff
@@ -14,3 +14,10 @@ def setUp(self):
             "1_MX4xMjM0NTZ-flNhdCBNYXIgMTUgMTQ6NDI6MjMgUERUIDIwMTR-MC40OTAxMzAyNX4"
         )
         self.opentok = OpenTok(self.api_key, self.api_secret)
+
+    def test_getset_appname_appversion(self):
```

Comment:
Will need to mod this test to take into account that `app_version` overrides the SDK version, and `app_name` is the custom 3rd chunk

Reply:
@dragonmantank :

The SDK version is `__version__` . So there is no way I am overriding that. If we take a look on the getters and setters for `app_name` and `app_version` you can see that i set those values on the protected attributes `_app_version` (NOT `__version__` , the SDK version) and `_app_name`.

If we check the headers method you will see that the sdk part for the user agent is fine

```
user_agent = (
    "OpenTok-Python-SDK/" + __version__ + " python/" + platform.python_version()
)
```

Then the application is going to concat the `_app_name` and the ` _app_version` at the end of the `user_agent`, just if both are different from null:

```
if self._app_name and self._app_version:
     user_agent += " {app_name}/{app_version}".format(
         app_name=self._app_name, app_version=self._app_version
     )
```

This action doesn't affect the SDK version in any way. Here an example:

```
>>> import opentok
>>> ot = opentok.OpenTok
>>> ot = opentok.OpenTok('a','b')
>>> ot.headers()[''User-Agent'']
```

```
>>> ot2 = opentok.OpenTok('c','d')
>>> ot2 = opentok.OpenTok('c','d',app_name='My App', app_version='1.0.0')
>>> ot2.headers()[''User-Agent'']
'OpenTok-Python-SDK/2.10.0 python/3.7.6 My App/1.0.0'
```

Opentok SDK version

```
>>> opentok.__version__
'2.10.0'
```